benchmarks/bm/iast_fixtures/str_methods.py

```diff
@@ -191,7 +185,7 @@ def do_translate(s, translate_dict):
     return s.translate(translate_dict)
 
 
-def do_decode_simple(s: bytes) -> str:
+def do_decode_simple(s: bytes) -> Text:
```

Comment:
I think for a decode test it's a lot better to check that the return value is explicitly str